test/coverage.runsettings

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+    <DataCollectionRunSettings>
+        <DataCollectors>
+            <DataCollector friendlyName="Code Coverage">
+                <Configuration>
+                    <SplitCoverage>True</SplitCoverage>
```

Comment:
what does SplitCoverage mean?

Reply:
SplitCoverage is so that we can see the Code Coverage tab in ADO.
More details from this comment here: https://github.com/microsoft/vstest/issues/981#issuecomment-839500012